_summaries/lang2018pointpillars_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: lang2018pointpillars
 # Please fill out info below
-author: # Your GitHub id
+author: aabayomi
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
-
+<!-- 
 TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Core Idea
+
+The authors proposes as much faster approach by which encoder that learns point cloud representations by stacking learned features as vertical tensors. 
+
+# Main Contributions
+
+![](lang2018pointpillars_2a.png)
+
+* Representation of the 3-D lidar point cloud inputs as pseudo-images
+
+* 2-D convolutional backbone processes the images
+
+* Single-shot detectors regresses to 3D boxes
+
+# Loss function 
```

Comment:
I like that you have discussed the loss function but it will be really good if you can elaborate on how the paper idea is realized technically like the three stages -> encoder conversion of point cloud to sparse pseudo images. 2D convolutional backbone network that takes the pseudo image and learns a high-level representation. And also on detection heads predicting boxes.

Reply:
You can also talk a bit about the inference settings like they used NMS.